src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1235,32 +1246,38 @@ internal static bool ContainsNativeIntegerWrapperType(this TypeWithAnnotations t
             return type.Type?.ContainsNativeIntegerWrapperType() == true;
         }
 
+        internal static bool ContainsErasedExtensionType(this TypeSymbol type)
```

Comment:
>ContainsErasedExtensionType [](http://example.com/codeflow?start=29&length=27)

The name feels somewhat misleading/confusing because , if the extension type is erased, then how it can be contained. Consider renaming to something like "ContainsExtensionTypeToErase"